st2common/st2common/services/workflows.py

```diff
@@ -351,14 +363,18 @@ def handle_action_execution_completion(ac_ex_db):
 
 
 def deserialize_conductor(wf_ex_db):
+    context = copy.deepcopy(wf_ex_db.flow['contexts'][0].get('__vars', {}))
+    context.get('st2', {}).update({'action_id': str(wf_ex_db.action_execution)})
```

Comment:
I would prefer we add a column in WorkflowExecutionDB for context and just serialize/deserialize from there instead of rebuilding this here.